ios/atb.xcodeproj/project.pbxproj

```diff
@@ -776,7 +776,7 @@
 				CODE_SIGN_STYLE = Manual;
 				CURRENT_PROJECT_VERSION = 1;
 				DEVELOPMENT_TEAM = "";
-				"DEVELOPMENT_TEAM[sdk=iphoneos*]" = CSV447Q97L;
+				"DEVELOPMENT_TEAM[sdk=iphoneos*]" = "$(IOS_DEVELOPMENT_TEAM_ID)";
```

Comment:
Wasn't it this that is replaced by fastline @toretefre ?

Reply:
Yes, but Fastlane does not help for local development, so this approach is better / complementary if it works

Reply:
@adriansberg is it verified with all orgs (atb/nfk)?

Reply:
I guess what's set for development team doesn't matter locally then?

Reply:
My previous comment was made with only seeing the first one somehow. Still somewhat relevant though, what does the development group do for local development? Have not verified it other than seeing that the values are set according to what's defined in the `.env` file.